lottie/src/main/java/com/airbnb/lottie/CompositionLayer.java

```diff
@@ -106,4 +108,16 @@ boolean hasMatte() {
     }
     return false;
   }
+
+  @Override
+  public void addColorFilter(@Nullable String layerName, @Nullable String contentName,
```

Comment:
i am not exactly sure what you are asking for here.

create another function such as `public void addColorFilter(@Nullable ColorFilter colorFilter)`?